File: iOS7_Primer/UITrial/UITrial/ViewController.m
//
//  ViewController.m
//  UITrial
//
//  Created by demo on 2014/06/17.
//  Copyright (c) 2014年 msyk.net. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@property (weak, nonatomic) IBOutlet UIImageView *imageView;

- (IBAction)tapButton:(id)sender;
- (IBAction)moveSlider:(id)sender;

@end

@implementation ViewController

- (IBAction)tapButton:(id)sender
{
    if ( self.imageView.alpha < 0.5 )    {
        self.imageView.alpha = 0.0;
    } else {
        self.imageView.alpha = 1.0;
    }
}

- (IBAction)moveSlider: (id)sender
{
    UISlider *slider = sender;
    self.imageView.alpha = slider.value;
}

- (void)viewDidLoad
{
    [super viewDidLoad];
	// Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning
{
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: iOS8_Primer_Swift/TrialUI/TrialUI/ViewController.swift
//
//  ViewController.swift
//  TrialUI
//
//  Created by demo on 2014/11/04.
//  Copyright (c) 2014年 msyk.net. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    @IBOutlet var imageView: UIImageView?
    
    @IBAction func tapButton(sender: AnyObject) {
        if imageView?.alpha < 0.5 {
            imageView?.alpha = 0.0
        } else {
            imageView?.alpha = 1.0
        }
    }
    
    @IBAction func moveSlider(sender: AnyObject)    {
        let slider = sender as UISlider
        imageView?.alpha = CGFloat(slider.value)
    }

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

